WordPress/Classes/ViewRelated/Site Creation/SiteIntent/SiteIntentData.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+
+struct SiteIntentData {
+    // Ordering of defaults affects ordering in the UI.
+    private static let defaultVerticalSlugs = [
+        "food",
+        "news",
+        "lifestyle",
+        "personal",
+        "photography",
+        "travel"
+    ]
+
+    static let verticals: Set<SiteIntentVertical> = [
```

Comment:
These were a lot more readable when I added tabs to align them, but the linter got angry.